config/__init__.py

```diff
@@ -0,0 +1 @@
+version = '1.0.1'
```

Comment:
1. no new line after this one. PEP8 issue
2. use underscores surrounding the word.

Reply:
Ok. Should i left this file or delete it. I mean, this variable i set up in ./__init__.py. So file config/__init__.py must be clean, as i understood. Am i right?